deviceModeInit.js

```diff
@@ -572,7 +639,7 @@ var rudderTracking = (function () {
   // rs$ = $.noConflict(true);
   // init();
   script.addEventListener("load", function () {
-    rs$ = $.noConflict(true);
```

Comment:
`noconflict()` allows other frameworks like Angular, Backbone etc which can also utilise '$' as a shortcut to be allowed to use '$', which otherwise would be held up by jquery hence that framework may stop working.

I think we should validate this case if we plan to remove this function.